src/SprykerEco/Yves/Payone/Plugin/PluginCountryFactory.php

```diff
@@ -14,16 +14,20 @@
 use SprykerEco\Yves\Payone\Plugin\SubFormsCreator\DefaultSubFormsCreator;
 use SprykerEco\Yves\Payone\Plugin\SubFormsCreator\DeSubFormsCreator;
 use SprykerEco\Yves\Payone\Plugin\SubFormsCreator\NlSubFormsCreator;
+use SprykerEco\Yves\Payone\Plugin\SubFormsCreator\SubFormsCreatorInterface;
 
 /**
  * @method \SprykerEco\Yves\Payone\PayoneFactory getFactory()
  */
 class PluginCountryFactory extends AbstractPlugin
 {
+    /**
+     * @var string
+     */
     public const DEFAULT_COUNTRY = 'default';
 
     /**
-     * @var \SprykerEco\Yves\Payone\Plugin\SubFormsCreator\SubFormsCreatorInterface[]
+     * @var array
```

Comment:
Fixed.

Reply:
Reverted. Because of PHPStan:
Property                                                               
|          SprykerEco\Yves\Payone\Plugin\PluginCountryFactory::$subFormsCreators  
|          (array<SprykerEco\Yves\Payone\Plugin\SubFormsCreator\SubFormsCreatorI  
|          nterface>) does not accept array<string,                               
|          (Closure)|(Closure)|(Closure)|(Closure)|(Closure)>